src/app/features/charts/components/score-chart/score-chart.component.ts

```diff
@@ -129,7 +131,7 @@ export class ScoreChartComponent extends BaseChartComponent {
       );
 
     // Colors
-    this.chart.colors(['#8b8ad1']);
+    this.chart.colors(this.barColor);
```

Comment:
Type seems to have changed from `string[]` to `string`, not sure if that matters. I don't see this property used anywhere.

Reply:
Yeah, the official docs mention an array but it didn't seem to affect the chart to use a string.  I will switch this to an array just to be safe.